set.mm

```diff
@@ -423483,8 +423483,8 @@ person who proved (or stated, in the case of a definition or axiom) the
        An exception should be made if a theorem is essentially an extract or a
        variant of an already existing theorem, in which case the contributor
        should be that of the statement from which it is derived, with the
-       modification signaled by a "(Revised by xxx, dd-Mmm-yyyy)" tag.  As with
-       all revision tags, it should be preceded by a short description of the
+       modification signaled by a "(Revised by xxx, dd-Mmm-yyyy.)" tag.  A
+       revision tag is optionally preceded by a short description of the
        revision.  Since this is somewhat subjective, judgment and intellectual
        honesty should be applied, with collegial settlement in case of dispute.
        </p>
```

Comment:
Since this paragraph is about "Contributors", I would suggest to move this sentence to the next paragraph "Usage of parentheticals.":

Usually, the comment of a theorem should contain at most one of the
       "Revised by" and "Proof shortened by" parentheticals, see Metamath Book,
       pp. 142-143 (there must always be a "Contributed by" parenthetical for
       every theorem).  Exceptions for "Proof shortened by" parentheticals
       are essential additional shortenings by a different person.  If a proof
       is shortened by the same person, the date within the "Proof shortened
       by" parenthetical should be updated only.  This also holds for "Revised
       by" parentheticals, except that also more than one of such
       parentheticals for the same person are acceptable (if there are good
       reasons for this).
       **A revision tag is optionally preceded by a short description of the
       revision.  Since this is somewhat subjective, judgment and intellectual
       honesty should be applied, with collegial settlement in case of dispute**.

Reply:
> would suggest to move this sentence to the next paragraph "Usage of parentheticals."

I agree that it is a bit awkward the way the current text is divided between those two sections so I guess moving those two sentences is OK with me.

To be honest I'm not really sure which concept "Since this is somewhat subjective, judgment and intellectual
honesty should be applied, with collegial settlement in case of dispute" was intended to modify. It seems to have been added in #2840 and I guess maybe it is about contribution comments in general? I'm not sure quite where to put it if we are moving text around.